.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version number to tag and release'
+        required: true
+        type: string
+      prerelease:
+        description: 'Release as pre-release'
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  release-npm:
+    runs-on: ubuntu-latest
+    environment: main
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+          cache: 'npm'
+          cache-dependency-path: package.json
+          check-latest: true
+
+      - name: Install
+        run: |
+          npm install --ignore-scripts
+
+      - name: Bump version
+        id: bump
+        run: |
+          new_version=$(npm version ${{ inputs.version }} --no-git-tag-version | tail -n 1)
+          echo "version=$new_version" >> $GITHUB_OUTPUT
+
+      - name: Commit version bump
+        run: |
+          git config --global user.name "${{ github.actor }}"
+          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
+          git commit -a -m "Bumped ${{ steps.bump.outputs.version }}"
+          git push origin HEAD:${{ github.ref }}
+
+      - name: Publish new version
+        run: |
+          npm publish --access public --tag ${{ inputs.prerelease == true && 'next' || 'latest' }}
+
+      - name: Create release notes
+        run: |
+          npx @matteo.collina/release-notes -a ${{ secrets.GITHUB_TOKEN }} -t ${{ steps.bump.outputs.version }} -r ${{ github.repository }} ${{ github.event.inputs.prerelease == 'true' && '-p' || '' }} -c ${{ github.ref }}
```

Comment:
not sure why we need this extra module?

we could use octokit directly: https://github.com/octokit/action.js/

this would to the trick:
- https://github.com/fastify/releasify/blob/main/lib/github.js#L38
and it supports already the [release note autogeneration](https://github.com/fastify/releasify/blob/093e476d237b82cce302e5e60b3cd4b583fb8539/lib/commands/publish.js#L183-L191)

Reply:
It was a suggestion by @mcollina. I will check the octokit one.

Reply:
@Eomm I took a look at releasefy, and it seems there's no way to generate only the Github release and considering we just need to call `npm publish` without a token from the command line, perhaps we simply don't need it?

I see it also bumps the package.json, but we'll still need to manually update fastify.js (`const VERSION =`), so getting the output from `npm version` would be easier.

Reply:
> I took a look at releasefy

I was showing the parameter only. Relesify is out of business here for sure

> I see it also bumps the package.json

We have a script now: 

`node build/sync-version.js`

https://github.com/fastify/fastify/commit/f15d4eaba03d86786bec2de7aed1236718c813f9